.pdm-python

```diff
@@ -0,0 +1 @@
+/home/kriese/src/renovate/.venv/bin/python
```

Comment:
Two things:

- The path points to your computer.
- End of file / end of line problem?